articles/java/sdk/java-sdk-azure-log.md

```diff
@@ -245,6 +245,16 @@ The following table shows the allowed values for this environment variable.
 |WARNING     |"warn", "warning"       |
 |ERROR    |"err", "error"  |
 
+## Setting an HttpLogDetailLevel
+Regardless of the logging mechanism used, if a client builder offers the ability to set [HttpLogOptions](https://docs.microsoft.com/en-us/java/api/com.azure.core.http.policy.httplogoptions?view=azure-java-stable), an [HttpLogDetailLevel](https://docs.microsoft.com/en-us/java/api/com.azure.core.http.policy.httplogdetaillevel?view=azure-java-stable) must also be specifed. This value defaults to `NONE`, so if it is not specified, no logs will be output even if the logging framework or default logging is properly configured. This value may be configured as follows.
+
+```
+new BlobClientBuilder().endpoint(<endpoint>)
+            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
```

Comment:
Do we also want to discuss the allowed headers and query parameters settings that need to be configured with the headers and query parameters that are allowed to be logged. Right now this will log the body but no headers or query parameters as none are set as being allowed to be logged.

Reply:
Yea I'll add that. I actually thought some were printed by default, but I'll update